dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -216,7 +220,8 @@ const sidePaddingDesktop = css`
 	}
 `;
 
-const accordionBottomMargin = css`
+const bodyWrapper = css`
```

Comment:
I slightly refactored this css to make it easier to adapt and to make the names a bit more truthful